nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/main/java/org/apache/nifi/controller/repository/StandardProcessSession.java

```diff
@@ -2527,6 +2554,43 @@ public String toString() {
         return "StandardProcessSession[id=" + sessionId + "]";
     }
 
+    @Override
+    public String getUnacknowledgedFlowfileInfo() {
```

Comment:
yeah i was just thinking that whenever rollback on a session is called it would write some details about the state of that session at that time to the log.